sys-tuner/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+authors = ["Solana Maintainers <maintainers@solana.com>"]
+edition = "2018"
+name = "solana-sys-tuner"
+description = "The solana cluster system tuner daemon"
+version = "0.22.0"
+repository = "https://github.com/solana-labs/solana"
+license = "Apache-2.0"
+homepage = "https://solana.com/"
+publish = true
+
+[dependencies]
+clap = "2.33.0"
+log = "0.4.8"
+semver = "0.9.0"
+solana-logger = { path = "../logger", version = "0.22.0" }
+unix_socket2 = "0.5.4"
```

Comment:
`unix_socker2` should be under `[target."cfg(unix)".dependencies]`, windows doesn't have it.

eg, https://github.com/solana-labs/solana/blob/5a30ef180abf0c87a01f2b17b5add4b5af7fe6ad/core/Cargo.toml#L74